source/set_up_3dsecure/index.html.md.erb

```diff
@@ -1,56 +1,103 @@
 ---
-title: Set up 3D Secure
+title: Set up Strong Customer Authentication
 last_reviewed_on: 2020-06-10
 review_in: 3 months
 weight: 170
 ---
 
-# Set up 3D Secure (3DS)
+# Strong Customer Authentication
 
-How you set up 3DS payment authentication depends on:
+Strong Customer Authentication (SCA) makes online payments more secure for users and reduces fraud. SCA is part of the Payment Service Directive (PSD2).
 
-- which Payment Service Provider (PSP) you're using
-- if you want to set up 3D Secure 2 (3DS2), which adds an extra layer of identity confirmation before your user pays
+You must implement SCA by:
+- XX December 2020 if you take payments from the European Economic Area (EEA), defined as the European Union, Norway, Iceland and Liechtenstein
+- XX September 2021 if you only take payments from within the UK
 
-## Stripe
+To implement SCA, you must set up 3D Secure 2 (3DS2) or 3D Secure (3DS) payment authentication.
 
-You do not need to do anything, because we set up 3DS2 on your service by default.
+3D Secure 2 (3DS2) adds an extra layer of identity confirmation before your user pays. You should implement 3DS2 if possible.
```

Comment:
we may not want to say it quite like that - in the product page we have. Maybe this nuance lives in the product page and not the tech docs?

3DS2 adds an extra layer of identity confirmation before the user pays so that transactions meet the SCA rules.

3DS2 replaces 3DS. 3DS transactions offer a similar level of protection for users and are compliant with SCA requirements, but introduce more friction to the user journey. 3DS will also be phased out, though the date for this has not been confirmed.

Reply:
More secure, less friction, 3DS1 is being phased out. Agreed to not endorse one or the other as this is not appropriate for this content.